api/apps/api/src/modules/projects/projects.controller.ts

```diff
@@ -288,8 +288,17 @@ export class ProjectsController {
   async updateBlmRange(
     @Param('id') id: string,
     @Body() { range }: UpdateProjectBlmRangeDTO,
+    @Req() req: RequestWithAuthenticatedUser,
   ): Promise<ProjectBlmValuesResponseDto> {
-    const result = await this.projectsService.updateBlmValues(id, range);
+    const result = await this.projectsService.updateBlmValues(
+      req.user.id,
+      id,
+      range,
+    );
+
+    if (!result) {
```

Comment:
This time it doesn't sound right - or can just make things confusing. Consider updating signature to return 
```
Either<UpdateBlmErrors, whateverItWasAsSuccess>

// where
type UpdateBlmErrors = typeof AclErrorSymbol | WhateverItWasAsError
```